adapter/test/data/hermes_ares.conf

```diff
@@ -0,0 +1,47 @@
+# Test configuration
+
+num_devices = 4;
+num_targets = 4;
+
+capacities_mb = {16384, 16384, 65536, 131072};
+block_sizes_kb = {4, 4, 4, 4};
+num_slabs = {4, 4, 4, 4};
+
+slab_unit_sizes = {
+  {1, 4, 16, 32},
+  {1, 4, 16, 32},
+  {1, 4, 16, 32},
+  {1, 4, 16, 32},
+};
+
+desired_slab_percentages = {
+  {0.25, 0.25, 0.25, 0.25},
+  {0.25, 0.25, 0.25, 0.25},
+  {0.25, 0.25, 0.25, 0.25},
+  {0.25, 0.25, 0.25, 0.25},
+};
+
+bandwidths_mbps = {6000, 4000, 2000, 200};
+latencies_us = {15, 250000, 500000, 1000000};
+
+buffer_pool_arena_percentage = 0.85;
+metadata_arena_percentage = 0.08;
+transfer_window_arena_percentage = 0.08;
+transient_arena_percentage = 0.03;
+
+max_buckets_per_node = 16;
+max_vbuckets_per_node = 8;
+system_view_state_update_interval_ms = 1000;
+
+mount_points = {"", "/dev/shm", "/mnt/nvme/hdevarajan/", "/mnt/nvme/hdevarajan/pvfs2-mount"};
+swap_mount = "/mnt/nvme/hdevarajan/pvfs2-mount";
```

Comment:
Can you remove your username from the test configuration?

Reply:
Is there a way to pass ENV variables? or should I use ./

Reply:
> Is there a way to pass ENV variables?

Not at the moment. I know it's a little annoying to have to change back and forth while testing. Just leave it as `./` for now and I'll think about something more convenient.

Reply:
I have used ./ for now.